hoodie-client/src/test/java/com/uber/hoodie/TestHoodieClientBase.java

```diff
@@ -107,6 +107,10 @@ protected HoodieWriteConfig getConfig() {
         .withIndexConfig(HoodieIndexConfig.newBuilder().withIndexType(HoodieIndex.IndexType.BLOOM).build());
   }
 
+  HoodieWriteConfig.Builder getConfigWithBulkInsertParallelismBuilder() {
+    return getConfigBuilder().withBulkInsertParallelism(1);
```

Comment:
1 is very restrictive.. Can we make it atleast 2, so there is some notion of parallel compute